addons/base_import/models/base_import.py

```diff
@@ -1294,21 +1292,20 @@ def _parse_import_data_recursive(self, model, prefix, data, import_fields, optio
                 # We should be able to manage both case
                 index = import_fields.index(name)
                 self._parse_float_from_data(data, index, name, options)
-            elif field['type'] == 'binary' and field.get('attachment') and any(f in name for f in IMAGE_FIELDS) and name in import_fields:
+            elif field['type'] == 'binary' and field.get('attachment') and name in import_fields:
                 index = import_fields.index(name)
 
                 with requests.Session() as session:
                     session.stream = True
 
                     for num, line in enumerate(data):
-                        if re.match(config.get("import_image_regex", DEFAULT_IMAGE_REGEX), line[index]):
+                        if re.match(config.get("import_image_regex", DEFAULT_URL_REGEX), line[index]):
```

Comment:
You mean ?
```suggestion
                        if re.match(config.get("import_url_regex", DEFAULT_URL_REGEX), line[index]):
```

IMO, you need to add an "deprecated option" in `_warn_deprecated_options()`.

Reply:
`_warn_deprecated_options()` seems to not take care of the unknown options as there are not defined in `config.py`.
> odoo.tools.config: unknown option 'import_image_maxbytes' in the config file at /tmp/.odoorc, option stored as-is, without parsing

Should we delete them or fix `_warn_deprecated_options` or i'm missing something ?